### PR TITLE
Gateway

### PR DESCRIPTION
**Pull request by gopalp1**

### CODE REVIEW

File: src/app/shared/components/data-filter-modal/data-filter-modal.component.html [hunk 1]
```diff
@@ -19,6 +19,6 @@
     </div>
 </div>
 <div mat-dialog-actions class="flex justify-between">
-    <button mat-button (click)="action()">Cancel </button>
-    <button mat-raised-button color="primary" (click)="action()">Submit</button>
+    <button mat-button class="close-button" (click)="action()">Cancel </button>
```
Comment (ecampusStreet) [inline src/app/shared/components/data-filter-modal/data-filter-modal.component.html:22]: Cancel button calling action method it should be call cancel button.

File: src/app/shared/components/data-modal/data-modal.component.html [hunk 2]
```diff
@@ -26,6 +26,6 @@ <h4>{{data?.title}}</h4>
     </div>
 </div>
 <div mat-dialog-actions class="flex justify-between">
-    <button mat-button (click)="action(data)">Cancel </button>
-    <button mat-raised-button color="primary" (click)="action(data)">Submit</button>
+    <button mat-button class="close-button" (click)="action(data)">Cancel </button>
```
Comment (ecampusStreet) [inline src/app/shared/components/data-modal/data-modal.component.html:29]: Cancel button calling action method it should be call cancel button.

File: src/app/admin/employee/add/add.component.html [hunk 3]
```diff
@@ -565,7 +565,7 @@ <h3>Present organization</h3>
           <div class="flex justify-between">
             <button mat-raised-button class="close-button" (click)="back()">Cancel</button>
             <div>
-              <button class="mx-2" mat-raised-button color="primary" *ngIf="employeeData?._id" (click)="getDocument(employeeData)" [disabled]="readOnly"> Documents</button>
+              <button class="mx-2 btn-button " mat-raised-button  *ngIf="employeeData?._id" (click)="getDocument(employeeData)" [disabled]="readOnly"> Letters</button>
```
[the remark this answers is not in the document]
Comment (gopalp1) [reply src/app/admin/employee/add/add.component.html:568]: yella kade label antha hakidvalla adakke

File: src/app/roles/list/list.component.html [hunk 4]
```diff
@@ -22,7 +22,7 @@
                           </tr>
                       </thead>
                       <tbody >
-                          <tr *ngFor="let role of list" (click)="action( role, 'view')">
+                          <tr *ngFor="let role of list" >
```
[the remark this answers is not in the document]
Comment (gopalp1) [reply src/app/roles/list/list.component.html:25]: work agatilla adakke remove madade

**Pull request merged**